prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -431,10 +445,8 @@ def prompt_processor() -> Any:
                         The date should be directly assignable to a date variable. \
                             If you cannot convert the string into a date, output "NA".'  # noqa
                 answer = run_completion(
-                    llm_helper=llm_helper,
-                    llm_li=llm_li,
-                    prompt=prompt,
-                    adapter_instance_id=adapter_instance_id,
+                    llm,
+                    prompt,
```

Comment:
@gaya3-zipstack Can we pass this as named args instead of positional args for better readability?

Reply:
Done